data/institutions/hauts_de_france.yml

```
@@ -1,3 +1,3 @@
-name: Haut de France
+name: Hauts de France
 imgSrc: img/logo_haut_de_france.svg
```

nitpick changer le nom du logo aussi :)
```suggestion
imgSrc: img/logo_hauts_de_france.svg
```

---

En effet ! Merci :)